src/worker_job.cc

```diff
@@ -75,6 +75,14 @@ namespace detail {
 
 	bool push_job::execute(const command_pkg& pkg, std::shared_ptr<logger> logger) {
 		if(data_handle == nullptr) {
+			// const celerity::detail::push_data
+			const auto data = boost::get<push_data>(pkg.data);
+			size_t elemSize = runtime::get_instance().getElemSizeOfBuffer(data.bid);
+			size_t numElems = data.sr.num_elems();
+			size_t bufTransferSize = elemSize * numElems;
+			logger->debug(logger_map(
```

Comment:
Please use the `trace` logging level here. We don't use `debug` anywhere else yet (and logging is a bit all of the place and in need of refactoring right now anyway, so we shouldn't add even more variations).

Reply:
Debug not being used anywhere else seems like a good reason to use it for this.